services/core-web/src/store/configureStore.ts

```diff
@@ -8,6 +8,14 @@ const getStore = (preloadedState = {}) =>
       ...rootReducer,
       loadingBar: loadingBarReducer,
     },
+    middleware: (getDefaultMiddleware) =>
+      getDefaultMiddleware({
+        // Disable check for immutable state as this has caused a lot of errors with the redux-toolkit introduction
+        // TODO: Consider fixing these issues and re-enabling this check
```

Comment:
I think that normally this just pops up as a dismiss-able error on development mode. Unless that's not the case I might prefer to keep it in so at least it tells you when you _add_ code that breaks those rules. Environment variables maybe? For the immutable especially, the serializable I think is a lost cause. (This is already an enormous PR so a hypothetical switch to env variables would be another ticket)

Reply:
Unfortunately it's shows up as a hard error here 😢 
<img width="1792" alt="image" src="https://github.com/bcgov/mds/assets/66635118/a66119f4-a715-413e-9298-c306b14b8467">